cvrf2csaf/section_handlers/vulnerability.py

```diff
@@ -91,6 +105,85 @@ def _handle_remediations(root_element):
 
         return remediations
 
+    @staticmethod
+    def _base_score_to_severity(base_score):
+        if base_score >= 9.0:
```

Comment:
Curiosity kills the 🐈 , but using the standard library module `bisect` this could be written less repetitive like so:
```python
        base_severity = ((0, 'NONE'), (0.1, 'LOW'), (4, 'MEDIUM'), (7, 'HIGH'), (9, 'CRITICAL'), (10, 'CRITICAL'))
        return base_severity[bisect.bisect_right(base_severity, (base_score, ''))][1]
```